test/integration_test/utils/storagecluster.go

```diff
@@ -235,17 +235,26 @@ func UpdateStorageCluster(cluster *corev1.StorageCluster) (*corev1.StorageCluste
 
 // UpdateAndValidateStorageCluster update, validate and return new StorageCluster
 func UpdateAndValidateStorageCluster(cluster *corev1.StorageCluster, f func(*corev1.StorageCluster) *corev1.StorageCluster, pxSpecImages map[string]string, t *testing.T) *corev1.StorageCluster {
-	liveCluster, err := operator.Instance().GetStorageCluster(cluster.Name, cluster.Namespace)
-	require.NoError(t, err)
+	var latestLiveCluster *corev1.StorageCluster
 
-	newCluster := f(liveCluster)
+	for i := 0; i < 5; i++ {
+		liveCluster, err := operator.Instance().GetStorageCluster(cluster.Name, cluster.Namespace)
+		require.NoError(t, err)
 
-	latestLiveCluster, err := UpdateStorageCluster(newCluster)
-	require.NoError(t, err)
+		newCluster := f(liveCluster)
 
-	logrus.Infof("Validate StorageCluster %s", latestLiveCluster.Name)
-	err = testutil.ValidateStorageCluster(pxSpecImages, latestLiveCluster, DefaultValidateUpdateTimeout, DefaultValidateUpdateRetryInterval, true, "")
-	require.NoError(t, err)
+		latestLiveCluster, err := UpdateStorageCluster(newCluster)
+		if err != nil && strings.Contains(err.Error(), k8sutil.UpdateRevisionConflictErr) {
+			logrus.Warnf("Failed to update StorageCluster [%s], Err: %s, Retrying..", latestLiveCluster.Name, err)
+			continue
+		}
+		require.NoError(t, err)
+
+		logrus.Infof("Validate StorageCluster %s", latestLiveCluster.Name)
+		err = testutil.ValidateStorageCluster(pxSpecImages, latestLiveCluster, DefaultValidateUpdateTimeout, DefaultValidateUpdateRetryInterval, true, "")
+		require.NoError(t, err)
```

Comment:
nit: This does not need to be in the for loop.

Reply:
Done